### DISCUSSION TITLE
Reload after update in modal

### DISCUSSION THREAD

**Discussion by MaartenJC** [Power-Components/livewire-powergrid#244] [Q&A] [answered]
Thank you for the great package!
I managed to implement PowerGrid with wire-elements/modal. So i want to update a record in a modal and when saving the form: 
1. update the record
2. close the modal
3. reload PowerGrid to show the updated record

1 and 2 are working. But how to reload PowerGrid?
Modal gives this option:
```
$this->closeModalWithEvents([

]);
```
What should i do to update the PowerGrid component?

**Comment by dansysanalyst** [comment]
Hi @mcouvreur,

I believe this is the case for a `pg:eventRefresh`.  You can read more here:

https://livewire-powergrid.com/#/table/component-settings?id=event-listeners

(This page was updated recently, it had a wrong syntax before).

**Reply by MaartenJC** [reply]
Thanks for your quick response.

I'm not sure how to implement this.
This is based on the example from Modal:

```
public function update()
{
    Gate::authorize('update', $user);
    
    $this->user->update($data);

    $this->closeModalWithEvents([
        'pg:eventRefresh-users'  => '$refresh',
    ]);
}
```

But this isn't working.

**Comment by dansysanalyst** [comment, marked as answer]
@luanfreitasdev has added an example today to https://github.com/Power-Components/powergrid-demo.

https://github.com/Power-Components/powergrid-demo/commit/c0702fe64079eb3196e2caf32b49e0f3742ea14e

Maybe it can help you :)

**Comment by MaartenJC** [comment]
Yes, i've got it working. Thanks a lot!
